ccl/importccl: TestImportPgDumpSchemas failed

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029702&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029702&tab=artifacts#/) on master @ [196130b8069778b7065cea3a3667ef07c41953ba](https://github.com/cockroachdb/cockroach/commits/196130b8069778b7065cea3a3667ef07c41953ba):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas727396398
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas727396398
--- FAIL: TestImportPgDumpSchemas (4.27s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210529 07:34:43.794829 524612 jobs/adopt.go:284  [n1] 1  job 662579718614941697: adoption completed with error job 662579718614941697: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 662579718614941697: cannot be reverted, manual cleanup may be required: descriptor not found
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (0.89s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029735&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029735&tab=artifacts#/) on master @ [196130b8069778b7065cea3a3667ef07c41953ba](https://github.com/cockroachdb/cockroach/commits/196130b8069778b7065cea3a3667ef07c41953ba):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas346161340
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas346161340
--- FAIL: TestImportPgDumpSchemas (14.75s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210529 12:37:01.822862 257562 jobs/adopt.go:284  [n1] 1  job 662639149991395329: adoption completed with error job 662639149991395329: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 662639149991395329: cannot be reverted, manual cleanup may be required: descriptor not found
E210529 12:37:01.844945 256673 jobs/registry.go:697  [-] 2  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210529 12:37:01.847753 256673 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.29s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029773&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029773&tab=artifacts#/) on master @ [196130b8069778b7065cea3a3667ef07c41953ba](https://github.com/cockroachdb/cockroach/commits/196130b8069778b7065cea3a3667ef07c41953ba):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas622679460
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas622679460
--- FAIL: TestImportPgDumpSchemas (15.73s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210529 18:28:09.370210 261681 jobs/adopt.go:284  [n1] 1  job 662708183090790401: adoption completed with error job 662708183090790401: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 662708183090790401: cannot be reverted, manual cleanup may be required: descriptor not found
E210529 18:28:09.391361 260780 jobs/registry.go:707  [-] 2  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.90s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029894&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029894&tab=artifacts#/) on master @ [7628160a7dfdc73d693f8018772fef8f79c8c0fe](https://github.com/cockroachdb/cockroach/commits/7628160a7dfdc73d693f8018772fef8f79c8c0fe):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas387389255
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas387389255
--- FAIL: TestImportPgDumpSchemas (2.90s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210529 21:27:38.623962 407307 jobs/adopt.go:284  [n1] 1  job 662743476220755969: adoption completed with error job 662743476220755969: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 662743476220755969: cannot be reverted, manual cleanup may be required: descriptor not found
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (0.90s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3030050&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3030050&tab=artifacts#/) on master @ [7628160a7dfdc73d693f8018772fef8f79c8c0fe](https://github.com/cockroachdb/cockroach/commits/7628160a7dfdc73d693f8018772fef8f79c8c0fe):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas077765252
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas077765252
--- FAIL: TestImportPgDumpSchemas (10.86s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210530 05:18:54.037937 226268 jobs/adopt.go:284  [n1] 1  job 662836126504288257: adoption completed with error job 662836126504288257: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 662836126504288257: cannot be reverted, manual cleanup may be required: descriptor not found
E210530 05:18:54.054279 225436 jobs/registry.go:697  [-] 2  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210530 05:18:54.056553 225436 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (2.75s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3031741&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3031741&tab=artifacts#/) on master @ [995f0dd1512d0266e78001980aab0023eab0e70f](https://github.com/cockroachdb/cockroach/commits/995f0dd1512d0266e78001980aab0023eab0e70f):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas235838748
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas235838748
--- FAIL: TestImportPgDumpSchemas (9.92s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210531 05:28:25.808367 223766 jobs/adopt.go:284  [n1] 5  job 663121116027944961: adoption completed with error job 663121116027944961: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663121116027944961: cannot be reverted, manual cleanup may be required: descriptor not found
E210531 05:28:25.822460 222754 jobs/registry.go:697  [-] 6  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210531 05:28:25.825142 222754 jobs/registry.go:707  [-] 7  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (2.45s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3032819&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3032819&tab=artifacts#/) on master @ [a64b26d37be6f781532dae4f8e4fc0cd189d075c](https://github.com/cockroachdb/cockroach/commits/a64b26d37be6f781532dae4f8e4fc0cd189d075c):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas604462232
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas604462232
--- FAIL: TestImportPgDumpSchemas (13.22s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210531 05:51:33.987278 264329 jobs/adopt.go:284  [n1] 1  job 663125663327420417: adoption completed with error job 663125663327420417: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663125663327420417: cannot be reverted, manual cleanup may be required: descriptor not found
E210531 05:51:34.000140 265231 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 2  making txn commit explicit failed for "unnamed" meta={id=3770de4e pri=0.00979379 epo=0 ts=1622440293.951603260,1 min=1622440293.931495863,0 seq=5} lock=true stat=STAGING rts=1622440293.951603260,1 wto=false gul=1622440294.431495863,0 int=1 ifw=4: result is ambiguous (server shutdown)
E210531 05:51:34.009361 263294 jobs/registry.go:697  [-] 3  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210531 05:51:34.012208 263294 jobs/registry.go:707  [-] 4  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.27s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3033102&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3033102&tab=artifacts#/) on master @ [b4791775a8801b0fee3625bbba9b0db28808371f](https://github.com/cockroachdb/cockroach/commits/b4791775a8801b0fee3625bbba9b0db28808371f):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas958124122
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas958124122
--- FAIL: TestImportPgDumpSchemas (14.37s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210531 11:14:35.824574 257407 jobs/adopt.go:284  [n1] 1  job 663189173233811457: adoption completed with error job 663189173233811457: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663189173233811457: cannot be reverted, manual cleanup may be required: descriptor not found
E210531 11:14:35.840873 258364 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 2  making txn commit explicit failed for "unnamed" meta={id=b521e692 pri=0.00436767 epo=0 ts=1622459675.787275165,0 min=1622459675.787275165,0 seq=5} lock=true stat=STAGING rts=1622459675.787275165,0 wto=false gul=1622459676.287275165,0 int=1 ifw=4: result is ambiguous (server shutdown)
E210531 11:14:35.846273 256574 jobs/registry.go:697  [-] 3  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210531 11:14:35.849003 256574 jobs/registry.go:707  [-] 4  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.55s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3033639&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3033639&tab=artifacts#/) on master @ [a3d8e3abb6172e91449eafe16f0143d493fc04b3](https://github.com/cockroachdb/cockroach/commits/a3d8e3abb6172e91449eafe16f0143d493fc04b3):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas389651550
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas389651550
--- FAIL: TestImportPgDumpSchemas (13.30s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210531 14:56:53.948298 253975 jobs/adopt.go:284  [n1] 2  job 663232880376250369: adoption completed with error job 663232880376250369: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663232880376250369: cannot be reverted, manual cleanup may be required: descriptor not found
E210531 14:56:53.974210 253009 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.40s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3034815&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3034815&tab=artifacts#/) on master @ [5fb0e29d1da5d09d1d869c2324d14160c0fbd71f](https://github.com/cockroachdb/cockroach/commits/5fb0e29d1da5d09d1d869c2324d14160c0fbd71f):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas841390445
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas841390445
--- FAIL: TestImportPgDumpSchemas (3.45s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 02:23:52.686329 509527 jobs/adopt.go:284  [n1] 1  job 663367949474660353: adoption completed with error job 663367949474660353: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663367949474660353: cannot be reverted, manual cleanup may be required: descriptor not found
E210601 02:23:52.720934 508695 jobs/registry.go:697  [-] 2  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210601 02:23:52.721306 508695 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (0.90s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035115&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035115&tab=artifacts#/) on master @ [7762f00fdb5dd7a65545b684fbd4f0babe383a76](https://github.com/cockroachdb/cockroach/commits/7762f00fdb5dd7a65545b684fbd4f0babe383a76):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas861154826
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas861154826
--- FAIL: TestImportPgDumpSchemas (9.55s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 05:20:14.740911 220864 jobs/adopt.go:284  [n1] 1  job 663402622566727681: adoption completed with error job 663402622566727681: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663402622566727681: cannot be reverted, manual cleanup may be required: descriptor not found
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (2.39s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035114&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035114&tab=artifacts#/) on master @ [7762f00fdb5dd7a65545b684fbd4f0babe383a76](https://github.com/cockroachdb/cockroach/commits/7762f00fdb5dd7a65545b684fbd4f0babe383a76):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas486181981
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas486181981
--- FAIL: TestImportPgDumpSchemas (1.81s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 05:33:59.477813 338645 jobs/adopt.go:284  [n1] 1  job 663405327722905601: adoption completed with error job 663405327722905601: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663405327722905601: cannot be reverted, manual cleanup may be required: descriptor not found
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (0.42s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3034924&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3034924&tab=artifacts#/) on master @ [5fb0e29d1da5d09d1d869c2324d14160c0fbd71f](https://github.com/cockroachdb/cockroach/commits/5fb0e29d1da5d09d1d869c2324d14160c0fbd71f):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas879818465
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas879818465
--- FAIL: TestImportPgDumpSchemas (12.31s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 03:57:34.498349 250526 jobs/adopt.go:284  [n1] 1  job 663386367769772033: adoption completed with error job 663386367769772033: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663386367769772033: cannot be reverted, manual cleanup may be required: descriptor not found
E210601 03:57:34.509514 251364 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 2  making txn commit explicit failed for "unnamed" meta={id=9f98cfaf pri=0.00111372 epo=0 ts=1622519854.438871964,0 min=1622519854.438871964,0 seq=3} lock=true stat=STAGING rts=1622519854.438871964,0 wto=false gul=1622519854.938871964,0 int=1 ifw=1: result is ambiguous (server shutdown)
E210601 03:57:34.517880 249542 jobs/registry.go:697  [-] 3  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210601 03:57:34.520559 249542 jobs/registry.go:707  [-] 4  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.08s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3036893&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3036893&tab=artifacts#/) on master @ [af3bebbcbdc8fe732fa4174afb2c93da2729c5c9](https://github.com/cockroachdb/cockroach/commits/af3bebbcbdc8fe732fa4174afb2c93da2729c5c9):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas204109313
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas204109313
--- FAIL: TestImportPgDumpSchemas (13.13s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 15:21:21.164038 258180 jobs/adopt.go:284  [n1] 1  job 663520803377086465: adoption completed with error job 663520803377086465: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663520803377086465: cannot be reverted, manual cleanup may be required: descriptor not found
E210601 15:21:21.181202 257151 jobs/registry.go:697  [-] 2  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210601 15:21:21.183408 257151 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.27s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3036875&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3036875&tab=artifacts#/) on master @ [82a4d1d40259eb10cafee551d10da4bf68b7302e](https://github.com/cockroachdb/cockroach/commits/82a4d1d40259eb10cafee551d10da4bf68b7302e):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas423784398
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas423784398
--- FAIL: TestImportPgDumpSchemas (13.68s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 15:16:27.944859 258307 jobs/adopt.go:284  [n1] 1  job 663519842870067201: adoption completed with error job 663519842870067201: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663519842870067201: cannot be reverted, manual cleanup may be required: descriptor not found
E210601 15:16:27.965265 257295 jobs/registry.go:697  [-] 2  error claiming jobs: could not query jobs table: claim-jobs: context canceled
E210601 15:16:27.969444 257295 jobs/registry.go:707  [-] 3  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.33s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/importccl.TestImportPgDumpSchemas [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3037453&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3037453&tab=artifacts#/) on master @ [82a4d1d40259eb10cafee551d10da4bf68b7302e](https://github.com/cockroachdb/cockroach/commits/82a4d1d40259eb10cafee551d10da4bf68b7302e):


```
=== RUN   TestImportPgDumpSchemas
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas923752835
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestImportPgDumpSchemas
    import_stmt_test.go:6293: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestImportPgDumpSchemas923752835
--- FAIL: TestImportPgDumpSchemas (13.53s)
=== RUN   TestImportPgDumpSchemas/inject-error-ensure-cleanup
E210601 18:25:48.370403 262017 jobs/adopt.go:284  [n1] 1  job 663557068556959745: adoption completed with error job 663557068556959745: cannot be reverted, manual cleanup may be required: descriptor not found
    import_stmt_test.go:6220: expected error 'testing injected failure', got: pq: job 663557068556959745: cannot be reverted, manual cleanup may be required: descriptor not found
E210601 18:25:48.393094 261192 jobs/registry.go:707  [-] 2  error processing claimed jobs: could not query for claimed jobs: select-running/get-claimed-jobs: context canceled
    --- FAIL: TestImportPgDumpSchemas/inject-error-ensure-cleanup (3.27s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestImportPgDumpSchemas PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportPgDumpSchemas.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -5996,7 +5996,6 @@ table_name FROM [SHOW TABLES] ORDER BY (schema_name, table_name)`,
 	})
 
 	t.Run("inject-error-ensure-cleanup", func(t *testing.T) {
-		skip.WithIssue(t, 65878, "flaky test")
 		defer gcjob.SetSmallMaxGCIntervalForTest()()
 		tc := testcluster.StartTestCluster(t, nodes, base.TestClusterArgs{ServerArgs: args})
 		defer tc.Stopper().Stop(ctx)

